styles/components/input.css

```diff
@@ -0,0 +1,14 @@
+:root{
+    --primary-color : #7C3AED;
+    --secondary-color : #F5F3FF;
+}
```

Comment:
Move all color variables in one single CSS file, instead of defining them in different files